felix/cmd/calico-bpf/commands/policy_debug.go

```diff
@@ -134,17 +126,22 @@ func dumpPolicyInfo(cmd *cobra.Command, iface, hook string) error {
 	if err != nil {
 		return err
 	}
+
 	cmd.Printf("IfaceName: %s\n", policyDbg.IfaceName)
 	cmd.Printf("Hook: %s\n", policyDbg.Hook)
 	cmd.Printf("Error: %s\n", policyDbg.Error)
 	cmd.Println("Policy Info:")
 	for _, insn := range policyDbg.PolicyInfo {
 		for _, comment := range insn.Comments {
 			if strings.Contains(comment, "Rule MatchID") {
-				count, err := getRuleCounter(comment)
+				count = 0
+				matchId, err := getRuleMatchID(comment)
 				if err == nil {
-					cmd.Printf("// count = %d\n", count)
+					if val, ok := m[matchId]; ok {
+						count = val
+					}
```

Comment:
what if it is not OK? Perhaps make a note if we found a counter that is not in the map. That could happen if the policy just changed and it got removed I think. Or not? Anyway, might be good not to be silent about this.

Reply:
Or you can just do `count = m[matchID]` and it would default to zero ...